sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/ScalaReflection.scala

```diff
@@ -179,6 +179,7 @@ trait ScalaReflection {
       case t if t <:< definitions.ShortTpe => Schema(ShortType, nullable = false)
       case t if t <:< definitions.ByteTpe => Schema(ByteType, nullable = false)
       case t if t <:< definitions.BooleanTpe => Schema(BooleanType, nullable = false)
+      case other => throw new UnsupportedOperationException(s"Schema for type $other is currently not supported")
```

Comment:
Seems like the DDOS on Github has it's affect on tests...
Tests failed due to

stdout: 
stderr: error:  while accessing https://github.com/apache/spark.git/info/refs
fatal: HTTP request failed

Hope it will be back online soon...